quinn/src/send_stream.rs

```diff
@@ -245,6 +252,25 @@ impl SendStream {
     }
 }
 
+/// Check if a send stream is stopped.
```

Comment:
Project convention is for the first line to not have a trailing period. This is private though so that's even more of a nitpick than usual.